src/Microsoft.ApplicationInsights/TelemetryClientExtensions.cs

```diff
@@ -176,5 +176,63 @@ public static IOperationHolder<T> StartOperation<T>(this TelemetryClient telemet
 
             operation.Dispose();
         }
+
+        /// <summary>
+        /// Creates an operation object with a respective telemetry item. 
```

Comment:
Please expand - explain whether Activity expected to be already started, which properties should already be initialized, etc.